app/Helpers/Helper.php

```diff
@@ -46,8 +46,9 @@ function durationToSpan($duration): string {
 function userTime(null|Carbon|\Carbon\Carbon|string $time=null, ?string $format=null, bool $iso=true): string {
     $format = $format ?? __('time-format');
     $time = $time instanceof \Carbon\Carbon ? $time : Carbon::parse($time);
+    $timezone = auth()->user()->timezone ?? config('app.timezone');
```

Comment:
timezone can't be null, because there is a db default

Reply:
can be null, because `auth()->user()` can be null.